lib/octicons_react/src/index.d.ts

```diff
@@ -6,6 +6,7 @@ type Size = 'small' | 'medium' | 'large'
 
 export interface OcticonProps {
   'aria-label'?: string
+  focusable?: boolean | 'auto'
```

Comment:
What's a good place to learn about `focusable="auto"`? I always thought it was a boolean value and would love to learn more 👀

Reply:
Oh, sure! I read it [here](https://www.w3.org/TR/SVGTiny12/interact.html#focus) Let me know if I miss anything!